### PR TITLE
feat: add bookmark fill icon

### PR DESCRIPTION
**Pull request by finalight**

### CODE REVIEW

**Comment by marthayaputra** [comment]
Lgtm

File: src/@next/Icon/icons/ri-bookmark-fill.svg
Comment (ninariccimarie) [inline src/@next/Icon/icons/ri-bookmark-fill.svg:1]: shouldn't this be `*-line`?

Comment (finalight) [reply src/@next/Icon/icons/ri-bookmark-fill.svg:1]: i got 2 variants, not sure how to go with this

image

Comment (ninariccimarie) [reply src/@next/Icon/icons/ri-bookmark-fill.svg:1]: usually the top one is fill and the bottom one is line

Comment (finalight) [reply src/@next/Icon/icons/ri-bookmark-fill.svg:1]: okay updated

**Review by ninariccimarie — APPROVED** [review]
LGTM

**Pull request merged**